interfaces/builtin/networkmanager.go

```diff
@@ -0,0 +1,409 @@
+				send_interface="org.freedesktop.NetworkManager.WiMax.Nsp"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.AccessPoint"/>
+
+		<!-- Devices (read-only, no security required) -->
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.Device.WiMax"/>
+
+		<!-- Devices (read/write, secured with PolicyKit) -->
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.Device.Wireless"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.Device"/>
+
+		<!-- Core stuff (read-only properties, no methods) -->
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.Connection.Active"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.DHCP4Config"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.DHCP6Config"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.IP4Config"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.IP6Config"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.VPN.Connection"/>
+
+		<!-- Core stuff (read/write, secured with PolicyKit) -->
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.Settings"/>
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.Settings.Connection"/>
+
+		<!-- Agents; secured with PolicyKit.  Any process can talk to
+				the AgentManager API, but only NetworkManager can talk
+				to the agents themselves. -->
+		<allow send_destination="org.freedesktop.NetworkManager"
+				send_interface="org.freedesktop.NetworkManager.AgentManager"/>
+		<deny send_interface="org.freedesktop.NetworkManager.SecretAgent"/>
+
+		<!-- Root-only functions -->
+		<deny send_interface="org.freedesktop.NetworkManager" send_member="SetLogging"/>
+		<deny send_interface="org.freedesktop.NetworkManager" send_member="Sleep"/>
+		<deny send_interface="org.freedesktop.NetworkManager.Settings" send_member="LoadConnections"/>
+		<deny send_interface="org.freedesktop.NetworkManager.Settings" send_member="ReloadConnections"/>
+		<deny send_interface="org.freedesktop.NetworkManager.VPN.Plugin"/>
+		<deny send_interface="org.freedesktop.NetworkManager.PPP"/>
+
+		<deny own="org.freedesktop.NetworkManager.dnsmasq"/>
+		<deny send_destination="org.freedesktop.NetworkManager.dnsmasq"/>
```

Comment:
This is a complex bus policy. What is its origin? Is it appropriate for Ubuntu Core? For example, Core has no way to prompt with policykit and these vpns aren't available.